.cirrus.yml

```diff
@@ -26,3 +26,21 @@ freebsd_task:
     - pw useradd cirrus -m
     - chown -R cirrus:cirrus .
     - sudo -u cirrus make test
+
+macos_task:
+  name: macOS m1
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+  only_if: $CIRRUS_TAG == ''
+  timeout_in: 20m
+  install_script:
+    - brew update
+    - brew install gettext libtool
+  build_script:
+    - NPROC=$(getconf _NPROCESSORS_ONLN)
+    - ./configure --with-features=${FEATURES}
+    - make -j${NPROC}
+  test_script:
+    - sudo sed -z 's/let buf = RunVimInTerminal('', {'rows': 6})/let buf = RunVimInTerminal('', {'rows': 10})/g' src/testdir/test_messages.vim
```

Comment:
I think our YAML parser is not happy here because of the `:`. You should probably extract it in a separate script with `"`:

```yaml
patch_script: "sudo sed -z 's/let buf = RunVimInTerminal('', {'rows': 6})/let buf = RunVimInTerminal('', {'rows': 10})/g' src/testdir/test_messages.vim"
```

Reply:
Thanks for hint :) https://github.com/vim/vim/pull/11226